scripts/request_withdraw.js

```diff
@@ -4,12 +4,18 @@ const { defaultWithdrawYargs, prepareWithdrawRequest } = require("./wrapper/with
 const { signAndExecute } = require("./utils/internals")(web3, artifacts)
 const { promptUser } = require("./utils/user_interface_helpers")
 
-const argv = defaultWithdrawYargs.option("noBalanceCheck", {
-  type: "boolean",
-  default: false,
-  describe:
-    "Request withdraw for tokens without checking if the token has no balance. Useful to account for trading that might be happening during the withdraw request",
-}).argv
+const argv = defaultWithdrawYargs
+  .option("stopTrading", {
+    type: "boolean",
+    default: false,
+    describe:
+      "Request withdraw of all funds on each bracket for each traded tokens, independently of their actual balance. Selected brackets will stop any trading from the following batch",
+  })
+  .option("onlySkipNonzero", {
```

Comment:
Are we sure this name shouldn't be `onlySkipZero`. Why would I want to skip non-zero balances? Either way, if we keep the name here I think the Z should be capitalized. Furthermore, in either case, `onlySkipZero`, I think this name could be reduced to `skipZero` (or `skipNonZero`) - i.e. without the only.

Reply:
I agree this is confusing, I changed it to `--skipZero`.
The "only" was there to point out that if you withdraw without `--skipZero` you are going to skip zero balances anyway, along with all balances of value <1$.